DEMO.md

```diff
@@ -0,0 +1,189 @@
+# DEMO
+
+The platform service is running on the endpoint specified in [this](https://docs.google.com/document/d/1C3s0ft4On8MIoi5v7dPXdsoNVCjLH8kMD2bdekHZ7Zg/edit?usp=sharing) gdoc (restricted to Red Hat organization). 
+
+First of all, export the base address of the Manager. For example, if the application is running on `localhost:8080` (replace this with the endpoint you extract from the link above), run
+
+```bash
+export MANAGER_URL=http://localhost:8080
+```
+
+## How to create a Bridge instance
+
+In order to send events to an Ingress, it is necessary to create a Bridge instance using the endpoint `/api/v1/bridges`. The request must include the name of the Bridge 
+
+```json
+{"name":  "myBridge"}
+```
+
+Run 
+
+```bash
+curl -X POST -H 'Accept: application/json' -H 'Content-Type: application/json' -d '{"name": "myBridge"}' $MANAGER_URL/api/v1/bridges
+```
+
+The response should look like something like
+
+```json
+{
+  "kind":"Bridge",
+  "id":"87508471-ee0f-4f53-b574-da8a61285986",
+  "name":"myBridge",
+  "href":"/api/v1/bridges/87508471-ee0f-4f53-b574-da8a61285986",
+  "submitted_at":"2021-09-24T11:29:33.086649+0000",
+  "status":"REQUESTED"
+}
+```
+
+Extract the Bridge ID and store it in another env variable (replace the value with the Bridge ID you get from the response) 
+
+```bash
+export BRIDGE_ID=87508471-ee0f-4f53-b574-da8a61285986
+```
+
+Until the Bridge is not in the `AVAILABLE` state, it is not possible to create Processors and to push events to the Ingress. 
+Check the status of the deployment with a GET request to the `/api/v1/bridges/{id}` endpoint: 
+
+```bash
+curl -X GET $MANAGER_URL/api/v1/bridges/$BRIDGE_ID
+```
+
+the response should look like 
+
+```json
+{
+  "kind":"Bridge",
+  "id":"87508471-ee0f-4f53-b574-da8a61285986",
+  "name":"myBridge",
+  "href":"/api/v1/bridges/87508471-ee0f-4f53-b574-da8a61285986",
+  "submitted_at":"2021-09-24T11:29:33.086649+0000",
+  "status":"AVAILABLE",
+  "endpoint":"/ingress/events/87508471-ee0f-4f53-b574-da8a61285986"
+}
+```
+
+The application is now `AVAILABLE` and we also have the information about the endpoint to use to push the events: `/ingress/events/87508471-ee0f-4f53-b574-da8a61285986` in this particular case.
+
+## How to add Processors to the Bridge
+
+It is possible to add Processors to the Bridge instance using the endpoint `/api/v1/bridges/{bridgeId}/processors`. 
+There are 3 basic concepts around a Processor:
+
+- **filters**: when one or more filters are specified, only those events that evaluates the filters to `true` will be processed by the specified action. The documentation around the filters can be found [here](FILTERS.md).
+- **transformations**: if a transformation schema has been specified in the request, the original event is transformed according to the schema before it is sent to the action.
+- **actions**: the action to take with the filtered events.
```

Comment:
I would prefer to switch the order: transformation concept is tied to action. Wdyt?
```suggestion
- **actions**: the action to take with the filtered events.
- **transformations**: an optional transformation logic can be specified for each action definition.
```